update.py

```diff
@@ -118,7 +118,6 @@ def main():
     args = parser.parse_args()
 
     platform = args.platform
```

Comment:
maybe let's keep it (it defaults x64_linux) so the check for the tarball presence is still in place for versions >= 4.2